Desktop/mainwindow.cpp

```diff
@@ -575,6 +575,22 @@ void MainWindow::setQmlImportPaths()
 	Log::log() << std::endl;
 }
 
+void MainWindow::generateWrappersForModule(QString moduleName)
+{
+	static bool alreadyDone = false; // The moduleInstallationSucceeded is called several times: this prevents to create several times the analyses.
```

Comment:
This will break if you install a development module `A`, uninstall it, and then install `B`...
Probably better to do a `static QString` with the modulename in there. 
Or something